### DISCUSSION TITLE
Left and right fingering position do not work well

### DISCUSSION THREAD

**Discussion by gerardoscoppetta** [opensheetmusicdisplay/opensheetmusicdisplay#1538] [Q&A] [answered]
I would like to point out that when I import an xmlmusic and render it as svg, if I set the fingering above or below it works well, however if the position to the left or right changes the original position.
I attach an example.
![test_fingering](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/41266915/02085809-5efc-4838-8538-495cdb8398e2)

**Comment by sschmidTU** [comment]
Can you please share the sample? I'm having a hard time reproducing this issue.
Are you sure the fingering is correct in the MusicXML?
<img width="135" alt="image" src="https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/33069673/a9cd1ac2-699b-460d-a6d2-92ff15d40ead">
settings:
```js
osmd.EngravingRules.FingeringPosition = 2;
osmd.EngravingRules.FingeringPositionFromXML = false;
```

**Reply by gerardoscoppetta** [reply]
Certain.
here is the same file I use to display the fingering on the left and the one above.
the fingering set above works, the one on the
[test_fingering.musicxml.zip](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/15307378/test_fingering.musicxml.zip)
 left fails in the first chord.
 I'll also post the two links where I did the test.
 https://www.suonosubito.net/test_opensheetmusicdisplay/test_fingering_above.php
 https://www.suonosubito.net/test_opensheetmusicdisplay/fingering_left.php

**Reply by sschmidTU** [reply, marked as answer]
Fixed. Thanks for the sample!
<img width="99" alt="image" src="https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/33069673/3484bcad-37c7-4ea2-8d6a-a01c92a75200">
I simplified the sample to one measure and added it to the tests:
[test_fingering_left_bach_short.zip](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/15311620/test_fingering_left_bach_short.zip)
